### PR TITLE
feat: doc for captions in js

### PR DESCRIPTION
**Pull request by amar-1995**

### CODE REVIEW

File: docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Captions (Transcription) (Beta)
```
Comment (mntrspace) [inline docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:2]: Live Transcription for Conferencing (Closed Captions - Beta)

File: docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+nav: 1.24
+---
+
+You can implement captions (subtitles) in your WEB conferencing app easily with 100ms SDK. The SDK provides a callback with transcript for each peer when they speak.
```
Comment (mntrspace) [inline docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:6]: 100ms' real-time transcription engine generates a live transcript (closed captions) during a conferencing session. The SDK provides a callback with transcript for each peer when they speak.

File: docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Minimum Requirements
+
+-   Minimum 100ms SDK version required is 0.10.8
+
+## What are captions?
+
+Captions are automatic subtitles that are generated with an AI model, and provided to your app with a easy to implement callback.
```
Comment (mntrspace) [inline docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:14]: You can remove this section.

File: docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    export interface useHMSTranscriptInput {
+        onTranscript?: (data: HMSTranscript[]) => void;
+        handleError?: hooksErrHandler;
+    }
+
+    export const useTranscript = ({ onTranscript, handleError = logErrorHandler }: useHMSTranscriptInput);
+}
+```
+
+### How to know if transcript is enabled in a room
```
Comment (mntrspace) [inline docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:43]: ## How do you know if closed captioning is enabled in a room?

Comment (mntrspace) [reply docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:43]: double ## only

File: docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Minimum Requirements
+
+-   Minimum 100ms SDK version required is 0.10.8
+
+## What are captions?
+
+Captions are automatic subtitles that are generated with an AI model, and provided to your app with a easy to implement callback.
+
+## How to implement captions?
```
Comment (mntrspace) [inline docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:16]: How to implement closed captioning?

**Review by mntrspace — CHANGES_REQUESTED** [review]

File: docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx (hunk 3 above)
Comment (mntrspace) [reply docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx:14]: Most people already know and we've already explained it above.

**Review by mntrspace — APPROVED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]

**Pull request merged**